iset.mm

```diff
@@ -117423,8 +117486,619 @@ Infinite sums (cont.)
       SIFYGYROUGABWSSYBVSVTYMYRYPSMRZAYQSYPMWBQYMYSASMRAYMYPASMAWCWDAWETWFTWHVB
       TWNYMXGXDOYOYMXGXCGKLZMRZXDYMGIFZXGUUAOWGABWSGYBVSVTYTGXCMWIQWJWHVBTWNABW
       SGYBWNWKUUBWNWGVDWLWM $.
+
+    $d A n $.
+    $( The exponential function of a positive real number is greater than 1
```

Comment:
just do a global search "exponential function of"

Reply:
I'm still not sure I'm sold on the idea that this is my job. But I did try making a pass through these (it wasn't always clear to me what to change), in set.mm and iset.mm.

Reply:
Well, it's no one's job, but we can try and fix the typos we run into.  Thanks for that pass.